web/admin.py

```
@@ -812,3 +821,141 @@ class PointsAdmin(admin.ModelAdmin):
+    list_filter = ("status", "rsvp_time", "check_in_time")
+    search_fields = ("event__title", "user__username", "user__email")
+    raw_id_fields = ("event", "user", "checked_in_by")
+    readonly_fields = ("rsvp_time", "check_in_time")
+    fieldsets = (
+        (None, {"fields": ("event", "user", "status")}),
+        ("Check-in", {"fields": ("check_in_time", "checked_in_by", "notes")}),
+        ("Timestamps", {"fields": ("rsvp_time",), "classes": ("collapse",)}),
+    )
+    actions = ["mark_as_present", "mark_as_no_show"]
+    
+    def mark_as_present(self, request: HttpRequest, queryset: QuerySet["ChapterEventAttendee"]) -> None:
```

Ruff (0.8.2)

Undefined name `HttpRequest`

(F821)